### ISSUE TITLE
Error in get_wavelengths for NIRISS SOSS exposure

### ISSUE THREAD

**Issue by hbushouse** [spacetelescope/jwst#3380]
When running the updated ``photom`` step on a NIRISS SOSS exposure, which now calls the ``get_wavelengths`` function contained in ``master_background.expand_to_2d`` to get wavelength values for all pixels, the following error is encountered:
```
2019-04-17 18:20:11,881 - stpipe.Spec2Pipeline.photom - INFO - PHOTMJSR value: 8.17066e-05
Traceback (most recent call last):
  File "/data1/jenkins/workspace/RT/JWST/jwst/pipeline/calwebb_spec2.py", line 91, in process
    asn.filename
  File "/data1/jenkins/workspace/RT/JWST/jwst/pipeline/calwebb_spec2.py", line 261, in process_exposure_product
    result = self.photom(input)
  File "/data1/jenkins/workspace/RT/JWST/jwst/stpipe/step.py", line 400, in run
    step_result = self.process(*args)
  File "/data1/jenkins/workspace/RT/JWST/jwst/photom/photom_step.py", line 51, in process
    result = phot.apply_photom(phot_filename, area_filename)
  File "/data1/jenkins/workspace/RT/JWST/jwst/photom/photom.py", line 808, in apply_photom
    self.calc_niriss(ftab)
  File "/data1/jenkins/workspace/RT/JWST/jwst/photom/photom.py", line 332, in calc_niriss
    self.photom_io(tabdata)
  File "/data1/jenkins/workspace/RT/JWST/jwst/photom/photom.py", line 665, in photom_io
    wl_array = expand_to_2d.get_wavelengths(self.input)
  File "/data1/jenkins/workspace/RT/JWST/jwst/master_background/expand_to_2d.py", line 336, in get_wavelengths
    wl_array = wcs(grid[1], grid[0])[2]
  File "/data1/jenkins/workspace/RT/JWST/miniconda/envs/tmp_env0/lib/python3.6/site-packages/gwcs/wcs.py", line 255, in __call__
    result = transform(*args, **kwargs)
TypeError: __call__() missing 1 required positional argument: 'spectral_order'
```

Apparently this is the first use of ``get_wavelengths`` on spectral data that has ``spectral_order`` as part of the wcs definition.

**Comment by philhodge** [comment]
We could assume that the spectral order is 1, but that's not really a good solution.

**Comment by hbushouse** [comment]
When this is called from the ``photom`` step, it's looping over spectral orders in the input data (to apply flux cal to each order), so it has an order number that could be passed as an additional argument to ``get_wavelengths``.

**Comment by hbushouse** [comment]
Correction to the above: for NIRISS SOSS it's not actually looping over orders, because they're all contained within a single image, but it is at least setting an order number that can be passed.

**Comment by hbushouse** [comment]
Fixed in #3387
